scripts/provisioning-tests

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+set -x
+
+# TODO: replace 1989024f0e122d6be84263021d9ad438ed7ce69e with HEAD~1
```

Comment:
I will address this todo comment once the PR has been approved. I am doing it this way to ensure the tests still run and the drone output is viewable and correct.